app/routes/wow.ilvl-shopping-list.tsx

```
@@ -0,0 +1,368 @@
+  name
+}: IlvlWoWListResponse & { sortby: string }) => {
+  useEffect(() => {
+    if (window && document) {
+      window.scroll({ top: 0, behavior: 'smooth' })
+    }
+  }, [])
+
+  return (
+    <PageWrapper>
+      <SmallTable
+        title={'Best Deals for ' + name}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)